### PR TITLE
Navigation Branch Complete

### PR DESCRIPTION
**Pull request by Noeyeddeer**

### CODE REVIEW

**Review by elysemontano — CHANGES_REQUESTED** [review]
How are your tests looking?

**Comment by Noeyeddeer** [comment]
Tests are looking much better now that I stopped ignoring them. I modified ApartmentIndex.test.js and Header.test.js and tested them both using yarn jest. Both tests passed.

File: app/javascript/components/components/Header.test.js [hunk 1]
```diff
@@ -14,9 +14,9 @@ import Header from './Header'
 Enzyme.configure({ adapter: new Adapter() })
 
 describe("When Header renders", () => {
-  it("displays a heading", () => {
+  it("displays four NavItems", () => {
     const header = shallow(<Header />)
-    const headerHeading = header.find("h1")
-    expect(headerHeading.text()).toEqual("This should fail")
+    const headerHeading = header.find("NavItem")
+    expect(headerHeading.length).toEqual(4)
```
Comment (elysemontano) [inline app/javascript/components/components/Header.test.js]: Will 4 NavItems always render on the page?  Does this test pass?

**Review by elysemontano — CHANGES_REQUESTED** [review]
One more thing to look at in Header.test.js

File: app/javascript/components/components/Header.test.js (hunk 1 above)
Comment (Noeyeddeer) [reply app/javascript/components/components/Header.test.js]: The test does pass but I could see where it wouldn't if I was logged in. I changed it to just look for the Nav and it passes.

File: app/javascript/components/components/Header.test.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ import Header from './Header'
 Enzyme.configure({ adapter: new Adapter() })
 
 describe("When Header renders", () => {
-  it("displays a heading", () => {
+  it("displays four NavItems", () => {
```
Comment (elysemontano) [inline app/javascript/components/components/Header.test.js]: Let's update this description to reflect the updated test.

**Review by elysemontano — CHANGES_REQUESTED** [review]
One last thing and you are good to go.  Make sure your test description reflects any updates in your test.

File: app/javascript/components/components/Header.test.js (hunk 2 above)
Comment (Noeyeddeer) [reply app/javascript/components/components/Header.test.js]: Updated to "displays one Nav"

**Review by elysemontano — APPROVED** [review]
Nice!

**Pull request merged**